src/runner.ts

```diff
@@ -101,7 +101,49 @@ export class ActionRunner {
       finalReport.usersToRequest = concatArraysUniquely(finalReport.usersToRequest, report.usersToRequest);
     }
 
-    return finalReport;
+    const { teamsToRequest, usersToRequest } = finalReport;
+    const reviewersLog = [
+      teamsToRequest ? `Teams: ${JSON.stringify(teamsToRequest)} - ` : "",
+      usersToRequest ? `Users: ${JSON.stringify(usersToRequest)}` : "",
+    ].join(" - ");
+
+    this.logger.info(`Need to request reviews from ${reviewersLog}`);
+
+    return await Promise.resolve();
```

Comment:
What does `return await Promise.resolve()` do?

Reply:
It's a residual. Good catch!